### PR TITLE
Fix show friends name on map #Cyclehack

### PR DESCRIPTION
**Pull request by huschu**
; disable colliding gesture recognizer in friend view

### CODE REVIEW

File: CriticalMass/ManageFriendsViewController.swift [hunk 1]
```diff
@@ -46,9 +46,10 @@ class ManageFriendsViewController: UIViewController, IBConstructable, UITableVie
         tableView.register(cellType: FriendTableViewCell.self)
         tableView.register(cellType: FriendSettingsTableViewCell.self)
         tableView.register(viewType: SettingsTableSectionHeader.self)
-        
-        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTap))
```
Comment (mltbnz) [inline CriticalMass/ManageFriendsViewController.swift]: What was this gesture for?

File: CriticalMass/BikeAnnotationView.swift [hunk 2]
```diff
@@ -40,17 +45,18 @@ class FriendAnnotationView: MKAnnotationView {
...
         canShowCallout = false
 
-        let friendView = FriendVie(frame: .init(x: 0, y: 0, width: 109, height: 23))
-        friendView.backgroundColor = .clear
-        friendView.name = friend?.name ?? ""
-        addSubview(friendView)
+        friendView = FriendView(frame: .init(x: 0, y: 0, width: 109, height: 23))
+        friendView!.backgroundColor = .clear
+        friendView!.name = friend?.name ?? ""
+        addSubview(friendView!)
```
Comment (mltbnz) [inline CriticalMass/BikeAnnotationView.swift]: Maybe we judt make the FriendView implicitly unwrapped so we can loose a couple of❗️

File: CriticalMass/BikeAnnotationView.swift [hunk 3]
```diff
@@ -40,17 +45,18 @@ class FriendAnnotationView: MKAnnotationView {
...
     }
 }
 
-class FriendVie: UIView {
+class FriendView: UIView {
```
Comment (mltbnz) [inline CriticalMass/BikeAnnotationView.swift]: 👍

File: CriticalMass/ManageFriendsViewController.swift (hunk 1 above)
Comment (huschu) [reply CriticalMass/ManageFriendsViewController.swift]: It should dismiss the keyboard when tapping anywhere in the table view but it broke the feature where tapping an online friend would focus the friend on the map.

**Pull request merged**